examples/sampling/node_classification.py

```diff
@@ -0,0 +1,284 @@
+"""
+This script trains and tests a GraphSAGE model for node classification on
+large graphs using efficient neighbor sampling.
+
+Paper: [Inductive Representation Learning on Large Graphs]
+(https://arxiv.org/abs/1706.02216)
+
+If you want a deeper understanding of node classification. You can
+read the example in the `examples/pytorch/graphsage/node_classification.py`
+TODO(#5797): Move `graphsage/node_classification.py` to the `examples/core/`.
+
+This flowchart describes the main functional sequence of the provided example.
```

Comment:
Do you still need this flowchart with the abstraction in `if __name__ == "__main__"`?

Reply:
I find it helpful since it has a  deeper than the main function.